docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+See the [database](databases.md#databases) section for implementation details on databases in simtools.
+
+Example:
+
+```bash
+simtools-add_model-parameters-from-repository-to-db \
+      --input_path /path/to/repository \
+      --db_name new_db_name
+```
+
+This application loops over all subdirectories in `input_path` and uploads all json files to the database `new_db_name` (or updates an existing database with the same name):
+
+- subdirectories starting with `OBS` are uploaded to the `sites` collection
+- json files from the subdirectory `configuration_sim_telarray` are uploaded to the `configuration_sim_telarray` collection
+- `Files` are added to the `files` collection
+- all other json files are uploaded to collection defined in the array element description in [simtools/schemas/array_elements.yml](https://github.com/gammasim/simtools/blob/main/simtools/schemas/array_elements.yml). Allowed values are e.g., `telescopes`, `calibration_devices`.
+
+## Import simulation model parameters
+
+### Import Prod6 model parameters
+
+Prod6 model parameters can be extracted from `sim_telarray` using the following commands.
+
+```bash
+./sim_telarray/bin/sim_telarray \
+   -c sim_telarray/cfg/CTA/CTA-PROD6-LaPalma.cfg \
+   -C limits=no-internal -C initlist=no-internal \
+   -C list=no-internal -C typelist=no-internal \
+   -C maximum_telescopes=30 -DNSB_AUTOSCALE \
+   -DNECTARCAM -DHYPER_LAYOUT -DNUM_TELESCOPES=30 \
+   /dev/null 2>|/dev/null | grep '(@cfg)'  >| all_telescope_config_la_palma.cfg
+
+./sim_telarray/bin/sim_telarray \
+   -c sim_telarray/cfg/CTA/CTA-PROD6-Paranal.cfg \
+   -C limits=no-internal -C initlist=no-internal \
+   -C list=no-internal -C typelist=no-internal \
+   -C maximum_telescopes=87 -DNSB_AUTOSCALE \
+   -DNECTARCAM -DHYPER_LAYOUT -DNUM_TELESCOPES=87 \
+   /dev/null 2>|/dev/null | grep '(@cfg)'  >| all_telescope_config_paranal.cfg
+```
+
+### Import Prod5 model parameters
+
+Prod5 model parameters can be extracted from `sim_telarray` using the following commands.
+
+```bash
+./sim_telarray/bin/sim_telarray \
+   -c sim_telarray/cfg/CTA/CTA-PROD5-LaPalma.cfg \
+   -C limits=no-internal -C initlist=no-internal \
+   -C list=no-internal -C typelist=no-internal \
+   -C maximum_telescopes=85 -DNSB_AUTOSCALE \
+   -DNECTARCAM -DHYPER_LAYOUT -DNUM_TELESCOPES=85 \
+   /dev/null 2>|/dev/null | grep '(@cfg)'  >| all_telescope_config_la_palma_prod5.cfg
+
+./sim_telarray/bin/sim_telarray \
+   -c sim_telarray/cfg/CTA/CTA-PROD5-Paranal.cfg \
+   -C limits=no-internal -C initlist=no-internal \
+   -C list=no-internal -C typelist=no-internal \
+   -C maximum_telescopes=120 -DNSB_AUTOSCALE \
+   -DNECTARCAM -DHYPER_LAYOUT -DNUM_TELESCOPES=120 \
+   /dev/null 2>|/dev/null | grep '(@cfg)'  >| all_telescope_config_paranal_prod5.cfg
+```
+
+### Import telescope positions
+
+Positions of array elements like telescopes are provided by CTAO in form of tables (typically ecsv files).
```

Comment:
Is there a path where we can find these files? Otherwise it looks like they are provided by e.g. email.

Reply:
We don't have a interfaces defined yet, so nobody knows how we get those files. So unfortunately we cannot be more precise at this point.